client/src/views/Login/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Button,
+  Form,
+  Grid,
+  Header,
+  Message,
+  Segment,
+} from 'semantic-ui-react';
+
+const Login = () => (
+  <Grid textAlign="center" style={{ height: '100vh' }} verticalAlign="middle">
+    <Grid.Column style={{ maxWidth: 450 }}>
+      <Header as="h1" color="teal" textAlign="center">
+        Posto
+      </Header>
+      <Header as="h2" color="teal" textAlign="center">
+        Log-in
+      </Header>
+      <Form size="large">
+        <Segment stacked>
+          <Form.Input
```

Comment:
Abstract these please, as these will be used at multiple places..

Reply:
Sure, will do it.